src/components/Products/Products.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import './Products.scss'
+import { Card, CardBody, CardFooter } from '@patternfly/react-core';
+
+import { NavLink } from 'react-router-dom';
+
+
+function Products(products) {
+    return (
+        <div className='products-container'>
+            {
+                products && products?.products?.map((product)=>{
+                    return(
+                        <NavLink to={`/product/${product.id}`} style={{textDecoration:"none"}} key={product.id}>
+                        <Card ouiaId="BasicCard">
+                            <CardBody>
+                                <img src={product.image} alt="" className='card-image'/>
```

Comment:
Add `alt` text to all the img tags you used also if possible, add width and height attributes as well.

Reply:
given meaningful alt text to all the images